build/dockerfiles/brew.Dockerfile

```diff
@@ -13,7 +13,7 @@
 # assumes you have created asset-*.tar.gz files for all arches, but you'll only unpack the one for your arch
 
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
-FROM ubi8-minimal:8.3-230
+FROM ubi8-minimal:8.4-205
 COPY asset-*.tar.gz /tmp/assets/
 RUN microdnf -y install tar gzip shadow-utils && \
```

Comment:
BTW why here we're not using multi-stage build ?

uncompress in builder image, then copy from builder to the runtime only the binary

Reply:
I suppose when Kirk wrote this, the thought was that because we've already built the binaries in Jenkins and collected them from the builder stage... there was no need to have a builder stage that just unpacks and copies.

But yeah we could ger a smaller runtime image if we do fewer steps and simply copies from the builder stage - so there's no need to install rpms and unpack / cleanup.